parquet-column/src/main/java/org/apache/parquet/filter2/predicate/Operators.java

```diff
@@ -247,6 +250,80 @@ public int hashCode() {
     }
   }
 
+  // base class for In and NotIn
```

Comment:
Have a better comment since it is public method

Reply:
Fixed. Thanks!